cmd/dlb_plugin/README.md

```diff
@@ -19,6 +19,85 @@ Table of Contents
 
 This Intel DLB device plugin provides support for [Intel DLB](https://builders.intel.com/docs/networkbuilders/SKU-343247-001US-queue-management-and-load-balancing-on-intel-architecture.pdf) devices under Kubernetes.
 
+### DLB2 driver configuration
+
+You should first build and load the dlb2 driver. After getting [DLB software release](https://downloadmirror.intel.com/690271/dlblinuxsrcrelease7.4.020211006.txz), build the kernel module and load the module as follows:
+```bash
+$ cd path/to/dlb/driver/dlb2/ && make
+$ sudo insmod dlb2.ko
+```
+
+Then, available dlb device nodes are visible in devfs.
+```bash
+$ ls -1 /dev/dlb*
+/dev/dlb0  /dev/dlb1  /dev/dlb2 ...
+```
+### VF configuration
+If you configure SRIO-V/VF (virtual functions), continue the following configurations.
```

Comment:
SR-IOV

Reply:
Oops, it's from the wiki. let me change both.!